library/src/main/java/com/getstream/sdk/chat/viewmodel/ChannelListViewModel.java

```diff
@@ -151,6 +155,50 @@ public void setChannelFilter(FilterObject filter) {
         this.filter = filter;
     }
 
+    /**
+     * hides the channel from queryChannels for the user until a message is added and remove from the current list of channels
+     *
+     * @param channel the channel needs to hide
+     */
+    public void hideChannel(@NotNull Channel channel) {
```

Comment:
probably good to add a callback handler in the signature to allow handling success / failure in the view